quinn-proto/src/connection.rs

```diff
@@ -1392,6 +1374,7 @@ where
                             self.in_flight.remove(&info);
                             self.space_mut(SpaceId::Data).pending += info.retransmits;
                         }
+                        self.streams.retransmit_all_for_0rtt();
```

Comment:
Is this new? I can't find the equivalent code in the old version.

Reply:
This used to be handled by the operation just above; `info.retransmits` contained sent stream frames from the packet in question, and `pending` contained the list of stream frames to send. With those removed in favor of the per-stream send buffers, we have to be more explicit.